gulpfile.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var gulp = require('gulp');
+var coffee = require('gulp-coffee')
+var gutil = require('gulp-util')
+var sourcemaps = require('gulp-sourcemaps')
+var del = require('del');
+
+gulp.task('dist', function() {
+  gulp.src([ 'src/**/*.coffee' ])
+  .pipe(coffee({bare: true}).on('error', gutil.log))
+  .pipe(gulp.dest('./'));
+});
+
+gulp.task('coffee', function() {
+  gulp.src('src/**/*.coffee')
+    .pipe(coffee({header: true}).on('error', gutil.log))
+    .pipe(gulp.dest('./'));
+});
+
+gulp.task('build', function() {
+  gulp.run('coffee');
+});
+
+gulp.task('default', function () {
+  gulp.run('build');
+
+  gulp.watch('src/**/*.coffee', function (event) {
+    gulp.run('coffee');
```

Comment:
`gulp.run` is deprecated. It's recommended to either switch to passing them in as an array or use `run-sequence`. For tasks that are order specific, I've had better success with [`run-sequence`](https://github.com/rentpath/tautline/blob/master/gulp/tasks/build.js#L5)